core/store/migrations/migrate_test.go

```diff
@@ -215,3 +217,28 @@ func TestMigrate_NewerVersionGuard(t *testing.T) {
 	})
 	require.NoError(t, err)
 }
+
+func TestMigration_1605816413(t *testing.T) {
+	t.Skip() // Slowish test, don't run in CI.
```

Comment:
I'm guessing this is probably far from the slowest test in the suite. Probably okay to keep around. Don't feel strongly though.

Reply:
yah I think its fine to skip it, seems quite difficult to introduce a migration regression so its probably not defending much by running in CI

Reply:
how slow is it?

Reply:
2.5s running by itself. That BootstrapThrowawayORM is pretty expensive and would probably slow other tests because they all run against the same db in CI

Reply:
That's nothing, I'd turn it on ;)